src/root/WithdrawManager.ts

```diff
@@ -119,6 +132,31 @@ export default class WithdrawManager extends ContractsBase {
     return this.web3Client.send(txObject, _options)
   }
 
+  async startExitWithBurntERC721TokensHermoine(burnTxHash, options?) {
+    const payload = await this.exitManager.buildPayloadForExitHermoine(
+      burnTxHash,
+      WithdrawManager.ERC721_WITHDRAW_EVENT_SIG
+    )
+    const txObject = this.erc721Predicate.methods.startExitWithBurntTokens(payload)
+    const _options = await this.web3Client.fillOptions(txObject, true /* onRootChain */, options)
+    if (_options.encodeAbi) {
+      return Object.assign(_options, { data: txObject.encodeABI(), to: this.erc721Predicate.options.address })
+    }
+    return this.web3Client.send(txObject, _options)
+  }
+
+  async getExitTime(burnTxHash, confirmTxHash) {
+    const HALF_EXIT_PERIOD = parseInt(await this.web3Client.call(this.withdrawManager.methods.HALF_EXIT_PERIOD()))
+    let blockNumber = (await this.web3Client.getParentWeb3().eth.getTransaction(confirmTxHash)).blockNumber
+    let confirmTime = (await this.web3Client.getParentWeb3().eth.getBlock(blockNumber)).timestamp
```

Comment:
transaction time and block time is same
no need of 2 separate calls, using transaction time also works

Reply:
what is transaction time ? 
getTransaction and getTransactionReceipt gives the blockNumber. it does not give transaction time. 
we need to use blockNumber to get the timestamp from getBlock. @ptsayli

Reply:
ya got it, `getTransaction` doesn't return a timestamp